Nika_modules/write_output.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+
+def save_filtered_fastq(filtered_seqs, input_path, output_filename=None):
```

Comment:
```suggestion
def write_fastq(fastq_data: dict, input_path, output_filename=None):
```
1) По аналогии с функцией для чтения
2) Ровно этому названию твоя функция и соотвествует
3) Я бы тут не упоминал ничего про filtered. Какая разница этой функции? Она вообще может быть никак с фильтром не связано. Ей какие FASTQ-данные дали, те она и записала. Так что тут можно более обще. 
4) Еще кстати аннотации типов не хватает:)